src/harrastuspassi/harrastuspassi/tests/fixtures_linkedcourses.py

```diff
@@ -62,7 +62,13 @@ def basic_event():
         },
         "location_extra_info": None,
         "description": {
-            "fi": "<p> <span style='background - color: initial;''>Työnjaossa pääset siivoamaan karsinoita ja tekemään muita navettatöitä, hoitamaan ja puuhailemaan erilaisten eläinten kanssa sekä ruokkimaan eläimiä. Voit esimerkiksi päästä taluttamaan lehmää, hypyttämään vuohta agility radalla, harjaamaan kania tai jopa ratsastamaan. Työnjako alkaa klo 10.30 ja loppuu klo 15. Ota mukaan säänmukaiset ja navettaan sopivat vaatteet sekä kumisaappaat, osa päivästä ollaan ulkona. Toimintapäiviin sisältyy yhteinen ruokatauko, jolloin syödään omia eväitä. </span> </p>"
+            "fi": "<p> <span style='background - color: initial;''>Työnjaossa pääset siivoamaan\
+            karsinoita ja tekemään muita navettatöitä, hoitamaan ja puuhailemaan erilaisten eläinten\
+            kanssa sekä ruokkimaan eläimiä. Voit esimerkiksi päästä taluttamaan lehmää, hypyttämään\
+            vuohta agility radalla, harjaamaan kania tai jopa ratsastamaan. Työnjako alkaa klo 10.30\
+            ja loppuu klo 15. Ota mukaan säänmukaiset ja navettaan sopivat vaatteet sekä kumisaappaat,\
+            osa päivästä ollaan ulkona. Toimintapäiviin sisältyy yhteinen ruokatauko, jolloin syödään\
+            omia eväitä. </span> </p>"
```

Comment:
This adds a lot of spaces inside the string (the indent is included). It doesn't matter here, but it can come up often with tests and long strings.

If you need to break long strings in python without newlines you can do this:

```python
foo = (
    "This is "
    "a long "
    "string"
)
```

```python
>>> foo
'This is a long string'
```

Reply:
true, that's a good way.